app/notification/application/map_contents.py

```diff
@@ -75,18 +75,28 @@ def get_questions_and_answers(data) -> dict:
                         NotifyConstants.APPLICATION_QUESTIONS_FIELD
                     ]:
                         for fields in question["fields"]:
-                            question_answers[form_name][
-                                fields["title"]
-                            ] = fields.get("answer")
+                            if fields["type"] == "file":
```

Comment:
⛏️ Please add some comments so this is more understandable?

Reply:
added

Reply:
Added!